sdk/lib/cpprt/arm/cpprt.s

```diff
@@ -15,9 +15,15 @@ _Target SETS "|$Target|"
     ; void __cdecl `eh vector constructor iterator'(void *,unsigned int,int,void (__cdecl*)(void *),void (__cdecl*)(void *))
     DEFINE_ALIAS ??_L@YAXPAXIHP6AX0@Z1@Z, ?MSVCRTEX_eh_vector_constructor_iterator@@YAXPAXIHP6AX0@Z1@Z
 
+    ; void __cdecl `eh vector constructor iterator'(void *,unsigned int,unsigned int,void (__cdecl*)(void *),void (__cdecl*)(void *))
+    DEFINE_ALIAS ??_L@YAXPAXIIP6AX0@Z1@Z, ?MSVCRTEX_eh_vector_constructor_iterator@@YAXPAXIHP6AX0@Z1@Z
```

Comment:
This line's second operand is the same as Line 16's second operand. Is there any evidence for this mapping?

Reply:
> Is there any evidence for this mapping (something dumped data)?

The 2 `MSVCRTEX` functions are implemented with `int` parameter only.
All i386/amd64/arm aliases map to them.
If that is wrong, then it is not specific to this PR.